styles/style.css

```diff
@@ -8,3 +8,90 @@ body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
 }
+
+#blue {
+  display: flex;
+  align-items: center;
+  background: linear-gradient(#3292e0, #1c5380);
+  padding-top: 10vw;
+  padding-bottom: 10vw;
+  padding-left: 50vw;
+}
+
+button {
+  background: linear-gradient(#3292e0, #1c5380);
+  border-radius: 2px;
+  margin-left: 40px;
+  padding: 5px 10px;
+  color: white;
+  font-weight: bold;
+  border: none;
+}
+
+button > img {
+  height: 11px;
+  padding-right: 5px;
+}
+
+article > img {
+  padding-right: 20px;
+  padding-top: 20px;
+}
+
+span {
+  font-weight: ;
+}
+
```

Comment:
After finishing the code, review and delete unused lines. It is very often you'll have leftover css while looking for the best style and experimenting :)